rust/main/agents/relayer/src/relayer.rs

```diff
@@ -520,8 +404,8 @@ impl BaseAgent for Relayer {
                 .get(dest_domain)
                 .and_then(|d| d.dispatcher_entrypoint.clone());
 
-            let db = match self.dbs.get(dest_domain) {
-                Some(db) => db.clone(),
+            let db = match self.origins.get(dest_domain) {
```

Comment:
Database is a weird case: it belongs to both origins and destinations. I'll add it to destination and this code will look better.

Reply:
okay :+1: